include/fiction/algorithms/simulation/sidb/time_to_solution.hpp

```diff
@@ -43,52 +64,64 @@ struct time_to_solution_stats
      * Average single simulation runtime in seconds.
      */
     double mean_single_runtime{};
-
     /**
      * Single simulation runtime of the exhaustive ground state searcher in seconds.
      */
     double single_runtime_exhaustive{};
-
+    /**
+     * Exhaustive simulation algorithm used to simulate the ground state as reference.
+     */
+    std::string algorithm;
     /**
      * Print the results to the given output stream.
      *
      * @param out Output stream.
      */
     void report(std::ostream& out = std::cout)
     {
-        out << fmt::format("[i] time_to_solution: {} | acc: {} | t_(s): {} | t_exhaustive(s): {}\n", time_to_solution,
-                           acc, mean_single_runtime, single_runtime_exhaustive);
+        out << fmt::format("[i] time_to_solution: {} | acc: {} | t_(s): {} | t_exhaustive(s): {} | exact alg.: {}\n",
+                           time_to_solution, acc, mean_single_runtime, single_runtime_exhaustive, algorithm);
     }
 };
 /**
  * This function determines the time-to-solution (TTS) and the accuracy (acc) of the *QuickSim* algorithm.
  *
  * @tparam Lyt Cell-level layout type.
  * @param lyt Layout that is used for the simulation.
- * @param sidb_params Physical SiDB parameters which are used for the simulation.
+ * @param quicksim_params Parameters required for the QuickSim algorithm.
```

Comment:
@marcelwa You are right! At the moment it is QuickSim by default. But since we have to add the physical parameters anyway, I thought it makes sense to use the quicksim_params directly as input.